Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -957,10 +966,40 @@ def get_incidents_command(client, args):
     )
 
 
+def check_if_incident_was_modified_in_xdr(incident_id):
+    demisto_incident = demisto.get_incidents()[0]
```

Comment:
move this call out of this function, pass here only the demisto_incident or `lastmirroredintime` which is better